libs/wingsdk/src/target-awscdk/function.ts

```diff
@@ -71,13 +72,48 @@ export class Function extends cloud.Function {
     super._bind(host, ops);
   }
 
+  /**
+   * Generates the code lines for the cloud function,
+   * overridden by the awscdk target to have the function context too
+   * @param inflightClient inflight client code
+   * @returns cloud function code string
+   * @internal
+   */
+  protected _generateLines(inflightClient: core.Code): string[] {
+    const lines = new Array<string>();
+
+    lines.push("exports.handler = async function(event, context) {");
+    lines.push(`
+  const console_log = console.log.bind({});
```

Comment:
no, it will reassign the function, and make a recursion...
the `.bind()` is used for cloning:
 https://stackoverflow.com/questions/1833588/javascript-clone-a-function

Reply:
TIL!